test/unit/contentful.test.ts

```diff
@@ -116,4 +116,16 @@ describe('contentful', () => {
       'http://some-base-url.com/environments/stage',
     )
   })
+
+  test('Initializes API with alpha features', () => {
```

Comment:
Not sure how useful this test is atm. 

What I'm missing so far in other tests is that none of explicitly checks if we are passing the correct query param. 

That's more useful than just checking if we are passing the `alphaFeatures` option down the chain.